ibmpisession/ibmpowersession.go

```diff
@@ -99,13 +166,28 @@ func New(iamtoken, region string, debug bool, timeout time.Duration, useraccount
 // NewAuth ...
 func NewAuth(sess *IBMPISession, PowerInstanceID string) runtime.ClientAuthInfoWriter {
 	var crndata = crnBuilder(PowerInstanceID, sess.UserAccount, sess.Region, sess.Zone)
+	var authdata = sess.IAMToken
 	return runtime.ClientAuthInfoWriterFunc(func(r runtime.ClientRequest, _ strfmt.Registry) error {
-		if err := r.SetHeaderParam("Authorization", sess.IAMToken); err != nil {
+
```

Comment:
Please move below code to a helper func and we need to also use it in `BearerTokenAndCRN` func as well.